File: laboratories/ppo/exercises/lab06/python/main.py
from src.ParkingLot import ParkingLot
from src.parking_vehicles.Bike import Bike
from src.parking_vehicles.Car import Car
from src.parking_vehicles.Truck import Truck


vehicles = [
    Bike(),
    Car(),
    Truck(),
    Car(),
]

parking_lot = ParkingLot()

for vehicle in vehicles:
    parking_lot.let_in(vehicle)

File: laboratories/ppo/exercises/lab06/java/Main.java
import lab06.ParkingLot;
import lab06.ParkingVehicles.Bike;
import lab06.ParkingVehicles.Car;
import lab06.ParkingVehicles.ParkingVehicle;
import lab06.ParkingVehicles.Truck;

public class Main {
  public static void main(String[] args) {
	ParkingVehicle[] vehicles = {
		new Bike(),
		new Car(),
		new Truck(),
		new Car(),
	};

	ParkingLot parkingLot = new ParkingLot();

	for(ParkingVehicle vehicle : vehicles) {
		parkingLot.letIn(vehicle);
	}
  }
}
